opt4j-optimizers/src/main/java/org/opt4j/optimizers/ea/aeseh/AeSeHModule.java

```diff
@@ -0,0 +1,261 @@
+		return epsilonNeighborhood;
+	}
+
+	public void setEpsilonNeighborhood(double epsilonNeighborhood) {
+		this.epsilonNeighborhood = epsilonNeighborhood;
+	}
+
+	public double getEpsilonNeighborhoodDelta() {
+		return epsilonNeighborhoodDelta;
+	}
+
+	public void setEpsilonNeighborhoodDelta(double epsilonNeighborhoodDelta) {
+		this.epsilonNeighborhoodDelta = epsilonNeighborhoodDelta;
+	}
+
+	public double getEpsilonNeighborhoodDeltaMax() {
+		return epsilonNeighborhoodDeltaMax;
+	}
+
+	public void setEpsilonNeighborhoodDeltaMax(double epsilonNeighborhoodDeltaMax) {
+		this.epsilonNeighborhoodDeltaMax = epsilonNeighborhoodDeltaMax;
+	}
+
+	public double getEpsilonNeighborhoodDeltaMin() {
+		return epsilonNeighborhoodDeltaMin;
+	}
+
+	public void setEpsilonNeighborhoodDeltaMin(double epsilonNeighborhoodDeltaMin) {
+		this.epsilonNeighborhoodDeltaMin = epsilonNeighborhoodDeltaMin;
+	}
+
+	/**
+	 * Returns the population size {@code alpha}.
+	 * 
+	 * @return the population size
+	 */
+	public int getPopulationSize() {
+		return populationSize;
+	}
+
+	/**
+	 * Sets the population size {@code alpha}.
+	 * 
+	 * @param alpha
+	 *            the population size to set
+	 */
+	public void setPopulationSize(int alpha) {
+		this.populationSize = alpha;
+	}
+
+	/**
+	 * Returns the number of generations.
+	 * 
+	 * @return the number of generations
+	 */
+	public int getGenerations() {
+		return generations;
+	}
+
+	/**
+	 * Sets the number of generations.
+	 * 
+	 * @param generations
+	 *            the number of generations
+	 */
+	public void setGenerations(int generations) {
+		this.generations = generations;
+	}
+
+	/**
+	 * Returns the number of children {@code lambda}.
+	 * 
+	 * @return the number of children
+	 */
+	public int getOffspringsPerGeneration() {
+		return offspringsPerGeneration;
+	}
+
+	/**
+	 * Sets the number of children {@code lambda}.
+	 * 
+	 * @param lambda
+	 *            the number of children
+	 */
+	public void setOffspringsPerGeneration(int lambda) {
+		this.offspringsPerGeneration = lambda;
+	}
+
+	/**
+	 * Returns the number of parents {@code mu}.
+	 * 
+	 * @return the number of parents
+	 */
+	public int getParentsPerGeneration() {
+		return parentsPerGeneration;
+	}
+
+	/**
+	 * Sets the number of parents {@code mu}.
+	 * 
+	 * @param mu
+	 *            the number of parents
+	 */
+	public void setParentsPerGeneration(int mu) {
+		this.parentsPerGeneration = mu;
+	}
+
+	/**
+	 * Returns the used crossover rate.
+	 * 
+	 * @return the crossoverRate
+	 */
+	public double getCrossoverRate() {
+		return crossoverRate;
+	}
+
+	/**
+	 * Sets the crossover rate.
+	 * 
+	 * @param crossoverRate
+	 *            the crossoverRate to set
+	 */
+	public void setCrossoverRate(double crossoverRate) {
+		this.crossoverRate = crossoverRate;
+	}
+
+	@Override
+	protected void config() {
+		bindIterativeOptimizer(EvolutionaryAlgorithm.class);
+		bind(CrossoverRate.class).to(ConstantCrossoverRate.class).in(SINGLETON);
```

Comment:
If we would remove line 256 and 257 we would have only one module for the general EA and this one for the aeseh selector and coupler.

Pro:
* This one module is a lot simpler without alpha, mu, and lambda.
* It would resemble the way we do it with Couplers in BasicMating and the Selectors NSGA, SMS, and SPEA
* You can freely combine selectors and couplers
Contra:
* You would need to select two modules, the standard EA and the AeSeh. We could, however, document this.